model/src/main/kotlin/config/OrtConfiguration.kt

```diff
@@ -41,6 +41,11 @@ data class OrtConfiguration(
      */
     val analyzer: AnalyzerConfiguration = AnalyzerConfiguration(),
 
+    /**
+     * The configuration of the downloader.
+     */
+    val downloader: DownloaderConfiguration? = null,
```

Comment:
The configurations for the other components are not (no longer) nullable. So this property should be initialized with a default configuration as well. This also simplifies code that accesses this value because no null checks are needed.